### PR TITLE
[About Us Page] Hero/Landing Component

### PR DESCRIPTION
**Pull request by daniel-panhead**
image

image

### CODE REVIEW

File: src/components/about-us/Hero.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import Image from "next/image";
```
Comment (IcedTea2K) [inline src/components/about-us/Hero.tsx]:
```suggestion
```

**Review by IcedTea2K — CHANGES_REQUESTED** [review]
LGTM! Just one-line nitpick

File: src/components/about-us/Hero.tsx (hunk 1 above)
Comment (daniel-panhead) [reply src/components/about-us/Hero.tsx]: Woooah u can suggest changes??

**Review by IcedTea2K — APPROVED** [review]
LGTM

**Pull request merged**